client-react/src/pages/app/app-settings/AppSettingsDataLoader.tsx

```diff
@@ -404,23 +407,28 @@ const AppSettingsDataLoader: React.FC<AppSettingsDataLoaderProps> = props => {
     return true;
   };
 
-  const errorPagesUpdated = (current: FormErrorPage[], origin: FormErrorPage[] | undefined) => {
-    let success = true;
+  const errorPagesUpdated = async (current: FormErrorPage[], origin: FormErrorPage[] | undefined) => {
```

Comment:
The functions in the two `forEach` statements are already `async`.

```suggestion
  const errorPagesUpdated = (current: FormErrorPage[], origin: FormErrorPage[] | undefined) => {
```

If you want `errorPagesUpdated` to block until all adds, deletes, and updates complete, consider using `Promise.all` instead of `Array.prototype.forEach`.